src/Compiler/CodeGen/IlxGen.fs

```diff
@@ -1229,6 +1240,12 @@ and IlxGenEnv =
 
         /// Indicates that the .locals init flag should be set on a method and all its nested methods and lambdas
         initLocals: bool
+
+        /// Delay code gen for files.
+        delayCodeGen: bool
+
+        /// Collection of code-gen functions where each function represents a file.
```

Comment:
It's a list of arrays, does each array belong to a file? (otherwise what do the arrays represent?)

Reply:
Yes, will improve the comment